service/src/test/java/io/camunda/service/TenantServiceTest.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.service;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertThrowsExactly;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import io.camunda.search.clients.TenantSearchClient;
+import io.camunda.search.entities.TenantEntity;
+import io.camunda.search.exception.NotFoundException;
+import io.camunda.search.filter.TenantFilter;
+import io.camunda.search.query.SearchQueryBuilders;
+import io.camunda.search.query.SearchQueryResult;
+import io.camunda.service.security.SecurityContextProvider;
+import io.camunda.zeebe.broker.client.api.BrokerClient;
+import java.util.List;
+import org.assertj.core.util.Arrays;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class TenantServiceTest {
```

Comment:
🔧 can we also add some tests in here related to the create method? An example of that test can be seen here https://github.com/camunda/camunda/blob/4dadab5b72bef05124abdcf87b68189552e8f378/service/src/test/java/io/camunda/service/RoleServiceTest.java#L105